gcloud/monitoring/label.py

```diff
@@ -48,7 +48,7 @@ class LabelDescriptor(object):
     :param description: A human-readable description for the label.
     """
 
-    def __init__(self, key, value_type, description):
+    def __init__(self, key, value_type=LabelValueType.STRING, description=''):
```

Comment:
This class looks like a good candidate for a namedtuple. The labeldescriptors are immutable objects, which need to support reading of attributes, and equality comparisons. We need to provide default values though, which can be done by overriding the classmethod `__new__()`.

A namedtuple would obviate the need for having all the instance methods below:
`_to_dict`, `__eq__`, `__ne__`, `__repr__`, and possibly `__hash__` in the future.